pyocd/target/family/target_imxrt.py

```diff
@@ -82,8 +84,19 @@ def set_reset_catch(self, reset_type=None):
         #              0001b - Parallel NOR boot via SEMC
         #              11xxb - Serial NAND boot via FlexSPI
         bootdevice = (self.read_memory(CortexM7_IMXRT.SRC_SBMR1) & 0x000000F0) >> 4
-        LOG.info("IMXRT Boot mode: %s" % self.BOOT_MODES[bootmode])
+        LOG.info("IMXRT Boot Device: %x" % bootdevice)
+        return bootdevice
+
+    def _get_flash_vector_addr(self):
+        mem = self.memory_map.get_boot_memory()
+        if mem and mem.type == MemoryType.FLASH:
+            return mem.start + 0x1004
+        return None
```

Comment:
The `return None` is unnecessary since None is the default return value from all functions.